simulators/ethereum/consensus/hivemodel.py

```diff
@@ -177,13 +177,17 @@ def perform_work(testcase):
                     testcase.details()
                 )
 
+        if True:
+            pool = ThreadPool(7) 
+            #Turns out a raw iterator isn't supported, so comprehending a list instead :(
+            pool.map(perform_work, [x for x in iterator()])
+            pool.close()
+            pool.join()
+        else:
```

Comment:
True, the point was just that I wanted to run it sequentially as a one-off hack, and be able to do it with minimal modification on the the server.